assets/js/components/Dashboard/PermanentConfiguration/AppBar.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ComponentType } from "react";
+import { CloseButton, Container, Navbar } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+
+interface AppBarProps {
+  title: string;
+}
+
+const Appbar: ComponentType<AppBarProps> = (props: AppBarProps) => {
+  const { title } = props;
+  const navigate = useNavigate();
+
+  return (
+    <Navbar variant="dark" data-bs-theme="dark">
+      <Container>
+        <Navbar.Brand>{title}</Navbar.Brand>
+        <CloseButton variant="white" onClick={() => navigate(-1)} />
```

Comment:
In the design, the color of the x looks the same as the "Configure screens" label, but not in implementation. Then I realized the implemented button has a hover color, which might be something do want to keep, but I'm not sure what the UI should be here, so maybe a q for Mary.

Reply:
Asked Mary what she thought [here](https://www.figma.com/file/FKWRVrSdoMjZOLorhuykvc?node-id=1760:58899&mode=design#618678226). After explaining to her how unusually complicated it is to change the color of the Bootstrap close button, she is ok with using its defaults.